.changeset/weak-files-admire.md

```diff
@@ -0,0 +1,5 @@
+---
+'@internal/plugin-todo-list': patch
```

Comment:
This package doesn't get released, it's a private example - so you can delete this changeset! 🙏